ratis-docs/src/site/markdown/configurations.md

```diff
@@ -675,4 +675,73 @@ For examples,
   2. However, setConf to a 3-member group by removing 2 of members and adding 2 new members is a majority-add.
 
 - Note also that adding 1 new member to an 1-member group is always allowed,
-  although it is a majority-add.
\ No newline at end of file
+  although it is a majority-add.
+
+
+## Client Configurations
+
+Client configurations are located at `RaftClientConfigKeys`.
+
+### RPC - Configurations related to Client RPC timeout.
+
+| **Property**    | `raft.client.rpc.request.timeout` |
+|:----------------|:----------------------------------|
+| **Description** | timeout for AppendEntries RPC     |
```

Comment:
How about `client side timeout for sending a request`?

Reply:
Good idea. Done